src/main/java/com/google/sps/data/FirebaseSingletonApp.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+
+/**
+ * Singleton class that will return the unique Firebase instance within the
+ * entire app.
+ */
+public class FirebaseSingletonApp {
+  private static FirebaseApp instance = initialize();
+
+  /**
+   * Initialize firebase SDK and return the FirebaseApp object
+   */
+
+  private static FirebaseApp initialize() {
+    try {
+      // System.out.println(System.getProperty("user.dir"));
+      FileInputStream serviceAccount = new FileInputStream(System.getProperty("user.home") + 
```

Comment:
This won't work when you try to deploy it.

Just use the recommended way: https://firebase.google.com/docs/admin/setup#initialize-sdk

When running locally set the environment variable to the correct location. When running on appengine, you don't need to do anything.

Reply:
Done